docs/guestfs.md

```diff
@@ -0,0 +1,165 @@
+# Libguestfs-tools 
```

Comment:
While I understand that the title is technically correct, because this is a user guide, what about choosing a user friendly name as th euser guide title, i.e. "Usage of libguestfs tools"?